tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -206,36 +258,53 @@ Create VCH - Existing VCH name
     Log To Console  Installer completed successfully: %{VCH-NAME}
 
     ${output}=  Run  bin/vic-machine-linux create --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=%{BRIDGE_NETWORK} --public-network=%{PUBLIC_NETWORK} --compute-resource=%{TEST_RESOURCE} ${vicmachinetls}
-    Should Contain  ${output}  already exists. Please delete it before reinstalling.
+    ${vm-folder-path}=  Run  govc ls | grep vm
+    Should Contain  ${output}  A VCH with the name \\"%{VCH-NAME}\\" already exists. Please delete it or choose a different VCH name before attempting reinstalling
```

Comment:
I'd shorten the contain so that's it's not SO dependent on the actual verbiage of the error message...i.e. `Should Contain ${output} "%{VCH-NAME}%" already exists`